### ISSUE TITLE
Enable fft  support for mobile builds

### ISSUE THREAD

**Issue by malfet** [pytorch/pytorch#62821]
Base on the conversation in https://github.com/pytorch/pytorch/pull/60976#issuecomment-893347471 it would be good to enable FFT operation support for PyTorchLite builds by default

**Comment by malfet** [comment]
I guess adding it as submodule to pytorch repo would automatically achieve this goal

**Comment by peterbell10** [comment]
Regarding https://github.com/pytorch/pytorch/pull/60976#discussion_r661753915, there is a pocketfft github at https://github.com/mreineck/pocketfft. The one listed in the comments of that PR is NOT the original author's github.

**Comment by malfet** [comment]
@peterbell10 I don't think we can use https://gitlab.mpcdf.mpg.de/mtr/pocketfft/ in our CI/CD, as the might not be accessible enough.

**Comment by hwangjeff** [comment]
hi @malfet, have there been any new developments on this? many of torchaudio's feature extraction utilities rely on fft — enabling fft support for mobile builds by default would facilitate wider adoption of torchaudio on mobile

**Comment by peterbell10** [comment]
FFT support for mobile should be fixed in #67909 which is in the latest 1.11.0 release.
